openff/toolkit/topology/molecule.py

```diff
@@ -6135,7 +6138,10 @@ def generate_unique_atom_names(self):
         with an atom name or type imported from another source. For example,
         generated atom names might begin 'C1x', 'H1x', 'O1x', 'C2x', etc.
         """
-        return _generate_unique_atom_names(self)
+        if suffix == None:
+            return _generate_unique_atom_names(self)
+        else:
+            return _generate_unique_atom_names(self, suffix)
```

Comment:
```suggestion
        return _generate_unique_atom_names(self, suffix)
```
And again, same as above